crates/runc-shim/src/synchronous/container.rs

```diff
@@ -21,6 +21,7 @@ use std::os::unix::io::{AsRawFd, FromRawFd};
 use std::path::Path;
 use std::sync::mpsc::{sync_channel, Receiver, SyncSender};
 
+use containerd_shim::protos::protobuf::EnumOrUnknown;
```

Comment:
should we make a rule of referencing packages order, like std/3rd/self

Reply:
I've fixed imports in https://github.com/containerd/rust-extensions/pull/95/commits/702d0212f4bf398c17a1d70c5bda28577fb19475
Can you elaborate on "make a rule"? I could not find the way to enforce imports order on CI :(

Reply:
Same with golang projects, I need to seperate these packages into three parts(std/3rd/self) by myself, the order of each part will be updated automatically by IDE or rust plugins. Thus, is it necessary to make demarcation? I also see many other repos they don't care about that.

Reply:
If it's troublesome, just let it go as it doesn't impact compile.